dev/devicelab/bin/tasks/plugin_dependencies_test.dart

```diff
@@ -274,13 +274,20 @@ public class DummyPluginAClass {
         section('Build plugin A example iOS app');
 
         await inDirectory(exampleApp, () async {
-          await evalFlutter(
+          final String output = await evalFlutter(
             'build',
             options: <String>[
               'ios',
               '--no-codesign',
+              '-v',
             ],
           );
+           // This warning is confusing and shouldn't be emitted. Plugins often support lower versions than the
+           // Flutter app, but as long as they support the minimum it will work.
+           // warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99. (in target 'plugin_a' from project 'Pods')
+           if (output.contains('but the range of supported deployment target versions')) {
```

Comment:
I may be wrong (hard to tell in GitHub), but it looks to me like lines 285-288 are indented 1 col too far, while 289-290 look ok?

Reply:
You're right.  Good eye (again).